### PR TITLE
Html scructure

### PR DESCRIPTION
**Pull request by DominicWebDev**
Please have a look !

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,7 +4,70 @@
...
+            width="89"
+        /></a>
+        <a href="https://www.youtube.com/watch?v=dQw4w9WgXcQ">
+          <img
+            src="https://cdn2.downdetector.com/static/uploads/c/300/de044/knuddels.logo.png"
+            alt="KnuddelsLogo"
+            height="60"
+            width="89"
+        /></a>
+      </h3>
```
Comment (FrankSiyli) [inline index.html:60]: /h3 should be in row 38 with some text

File: index.html [hunk 2]
```diff
@@ -4,7 +4,70 @@
...
+      <h3>
+        <p id="Skills">Skills</p>
+      </h3>
```
Comment (FrankSiyli) [inline index.html:62]: no paragraph tag needed between headline tags

File: index.html [hunk 3]
```diff
@@ -4,7 +4,70 @@
...
+      <ul>
+        <li>Java</li>
+        <li>Html</li>
+        <li>React</li>
+        <li>JavaScript</li>
+      </ul>
+      <h5><footer>Thanks for your time</footer></h5>
```
Comment (FrankSiyli) [inline index.html]: arrangement ; footer outside and headline inside should be correct

**Review by FrankSiyli — COMMENTED** [review]
rest LGTM👌

**Review by Satttoshi — APPROVED** [review]
after @FrankSiyli suggestions, i would say LGTM 🚀 
Site looks nice tho ✌️

**Review by manukao — APPROVED** [review]
awesome, lgtm 👍

**Comment by toshydev** [comment]
Very nice! 
LGTM👌

**Pull request merged**